packages/Liquid-Core.package/LiquidHostMenu.class/instance/startPoll.st

```diff
@@ -18,6 +18,9 @@ pollNameInput := FillInTheBlank request: 'Name your Poll'.
 			newPollDraft questionList: self questionList.
 			newPoll := LiquidPoll newWithPollDraft: newPollDraft.
 			newPoll id: pollNameInput.
+			newPoll open: true.
```

Comment:
[Boulder] The requirement to open polls when creating them likely makes the CSV test fail. Either add open: true to the test, or set open to true in an initialize method. I highly recommend setting a value to open in initialize to make things clear either way.  (I lean towards setting it to true, but defaulting to closed could theoretically be more secure).
[Sand] Also, we should really start to use cascading more (this ain't your fault :) ).